daemon/containerd/image_list.go

```diff
@@ -285,6 +289,23 @@ func (i *ImageService) singlePlatformImage(ctx context.Context, contentStore con
 		Containers: -1,
 	}
 
+	if opts.ContainerCount {
+		// Lazily init allContainers.
+		if allContainers == nil {
```

Comment:
Silly question, in what case would allContainers _not_ be nil here? Because it looks like it's initialised at line 274?

Reply:
Not a silly question, this is not at all what I wanted to do, fixed this, I'm loading the containers in the caller now when needed